.github/workflows/site-doc.yml

```diff
@@ -0,0 +1,39 @@
+name: Build Website
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "docs/**"
+      - '.github/workflows/site-doc.yml' # This workflow
```

Comment:
can you combine this with https://github.com/meta-llama/llama-stack/blob/main/.github/workflows/update-readthedocs.yml so that the full workflow:
- does a build check like your workflow does
- once that succeeds, it triggers a build on readthedocs?

Reply:
Done!